contracts/test/SafeMock.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
It can be removed, already included In upgrade-tools

Reply:
It has to be removed.